azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/service/calling/CallingService.kt

```diff
@@ -43,6 +44,12 @@ internal class CallingService(
         * Expected behavior.
         * */
         private const val CALL_END_REASON_DECLINED = 603
+        internal const val CALL_END_REASON_EVICTED = 5000
```

Comment:
any reason to diverge here with internal instead of private? 

Since it's a library of error codes, I think making them all internal is OK here for consistency.

Reply:
I needed it exposed as internal for writing the Unit Tests on call eviction.  Which kinda says we don't have Unit tests for  CALL_END_REASON_TOKEN_EXPIRED, CALL_END_REASON_DECLINED, CALL_END_REASON_CANCELED.  I'll change the other constants to internal for consistency.
@Mohtasim can you write up child tickets for the Unit test user story that you created ?